src/jit/codegenxarch.cpp

```diff
@@ -6827,6 +6860,12 @@ void        CodeGen::genCompareLong(GenTreePtr  treeNode)
     assert(varTypeIsLong(op1->TypeGet()) && varTypeIsLong(op2->TypeGet()));
```

Comment:
Since you are here, please consider splitting this assert into two asserts

e.g.
```
assert(varTypeIsLong(op1));
assert(varTypeIsLong(op2));
```